### PR TITLE
[JFK-DAM]  Correct stack errors and rails interop errors

### PR DESCRIPTION
**Pull request by sdbradley**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,21 @@
 source 'https://rubygems.org'
 
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
 # Provides basic authentication functionality for testing parts of your engine
-gem 'solidus', github: 'solidusio/solidus'
+gem 'solidus', github: 'geminimvp/solidus', branch: 'image_overrides'
```
Comment (fastjames) [inline Gemfile]: I think the branch you want is `gemini_master` as that should have the overrides plus one or two other minor features we needed.

**Review by fastjames — APPROVED** [review]
Glad to see this and hear that specs are :green_heart:.

**Comment by sdbradley** [comment]
yes, :green_heart: is good!   😄

**Comment by sdbradley** [comment]
updated `solidus` branch to `gemini_master`, still :green_heart:

**Comment by fastjames** [comment]
I have added CircleCI, squashed this, and verified all green so I'm going to merge it.

**Pull request merged**